R/GetReport.R

```diff
@@ -26,13 +27,14 @@
 #' @export
 #'
 
-GetReport <- function(report.id,interval.seconds=10,max.attempts=3,print.attempts=TRUE) {
+GetReport <- function(report.id,interval.seconds=10,max.attempts=3,print.attempts=TRUE,format="json") {
```

Comment:
Would that mean duplicating some of the functionality of GetReport inside QueueDatawarehouse?